spec/lib/hackney/income/stored_tenancies_gateway_spec.rb

```diff
@@ -393,23 +393,29 @@
       context 'with is_paused set false' do
         let(:is_paused) { false }
 
-        it 'shows the number pages of of paused cases' do
+        it 'shows the number pages of paused cases' do
```

Comment:
Should this be 
```suggestion
        it 'shows the number of pages of paused cases' do
```
?